docs/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration.md

```diff
@@ -183,6 +183,16 @@ Clusters that were created before Kubernetes 1.16 will have an `ingress-nginx` `
 
 If the `updateStrategy` of `ingress-nginx` is `OnDelete`, you will need to delete these pods to get the correct version for your deployment.
 
+### Cluster Agent Configuration and Fleet Agent Configuration
+
+You can customize the scheduling fields and resource limits for the cluster agent, and for the cluster's Fleet agent. You can use this feature to customize cluster tolerations, affinity rules, and resource requirments. Your customizations will override pre-existing rules and default settings.
```

Comment:
Suggestion (fix wording to make the functionality more clear and spelling)

`You can customize the scheduling fields and resource limits for the Cluster Agent and the cluster's Fleet Agent. You can use this feature to customize tolerations, affinity rules, and resource requirements. Additional tolerations will be appended to a list of default tolerations and control plane node taints. Defining custom affinity rules will override the global default affinity setting, and defining resource requirements will set requests or limits where there was none.`